src/pages/Admin/Venue/AnnouncementStatus/AnnouncementStatus.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { Banner } from "types/banner";
+
+import { isDefined } from "utils/types";
+
+import "./AnnouncementStatus.scss";
+
+export interface AnnouncementStatusProps {
+  banner?: Banner;
+}
+
+const ENABLED__TEXT = "enabled";
+const DISABLED__TEXT = "disabled";
+
+export const AnnouncementStatus: React.FC<AnnouncementStatusProps> = ({
+  banner,
+}) => {
+  const fullscreenImg = banner?.isFullScreen ? faCheck : faTimes;
+  const closeImg = banner?.isForceFunnel ? faCheck : faTimes;
+
+  const fullscreenText = banner?.isFullScreen ? ENABLED__TEXT : DISABLED__TEXT;
+  const forceText = banner?.isForceFunnel ? ENABLED__TEXT : DISABLED__TEXT;
```

Comment:
looks like we have duplicated logic here. Can we create 2 functions and move them out of the component.
```ts
const getStatusImage = (status) => status ? faCheck : faTimes;
const getStatusText = (status) => status ? ENABLED__TEXT : DISABLED__TEXT;
```

the names of the functions and arguments can be improved. 🙃

Reply:
done ed799a1